experimental/torch_xla2/torch_xla2/ops/jaten.py

```diff
@@ -1562,8 +1568,12 @@ def _aten_as_strided_scatter(x, src, sizes, strides, storage_offset):
 
 # aten.atan2
 @op(torch.ops.aten.atan2)
-def _aten_atan2(self, other):
-  return jnp.arctan2(self, other)
+def _aten_atan2(input, other):
```

Comment:
can use `_handle_int64_trig` like this: https://github.com/pytorch/xla/blob/master/experimental/torch_xla2/torch_xla2/ops/jaten.py#L881

Reply:
Thanks, updated with `_handle_int64_to_int32_trig`